odoo_office365_contacts==UPDATED/odoo_office365_contacts/models/__init__.py

```
@@ -0,0 +1,4 @@
+# See LICENSE file for full copyright and licensing details.
+from . import office365lib
```

Ruff (0.11.9)

`.office365lib` imported but unused; consider removing, adding to `__all__`, or using a redundant alias

(F401)